resolver-dns/src/main/java/io/netty/resolver/dns/DnsServerAddressStreamProviders.java

```diff
@@ -25,7 +26,10 @@
     // TODO(scott): how is this done on Windows? This may require a JNI call to GetNetworkParams
     // https://msdn.microsoft.com/en-us/library/aa365968(VS.85).aspx.
     private static final DnsServerAddressStreamProvider DEFAULT_DNS_SERVER_ADDRESS_STREAM_PROVIDER =
-            UnixResolverDnsServerAddressStreamProvider.parseSilently();
+            // If on windows just use the DefaultDnsServerAddressStreamProvider.INSTANCE as otherwise
+            // we will log some error which may be confusing.
+            PlatformDependent.isWindows() ? DefaultDnsServerAddressStreamProvider.INSTANCE :
```

Comment:
If the issue is "the logs are confusing" ... then should we either:

- not log in `DnsServerAddressStreamProvider#parseSilently()` if on windows? 
- make parseSilently more strict and just throw if it fails ... and handle the errors at this level because we are introducing conditionals based on platform here

Reply:
honestly I think we should not try to call something called `Unix*` on windows, thats why I did the change here.

Reply:
makes sense ... I guess it still makes sense for parseSilently to tolerate a failure on unix based systems too ... in case the system configuration is not as expected.

Reply:
+1